apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/backend/codegen/EnumType.kt

```diff
@@ -48,7 +51,10 @@ private fun CodeGenerationAst.EnumType.toEnumTypeSpec(generateAsInternal: Boolea
 }
 
 private fun CodeGenerationAst.EnumType.adapterTypeSpec(generateAsInternal: Boolean, asSealedClass: Boolean, packageName: String): TypeSpec {
-  val fromResponseFunSpec = fromResponseFunSpecBuilder()
+  val fromResponseFunSpec = FunSpec.builder("fromResponse")
+      .addModifiers(KModifier.OVERRIDE)
+      .addParameter(ParameterSpec.builder("reader", JsonReader::class).build())
```

Comment:
nit: should we use `Identifiers.reader`?

Reply:
Yup, make sense.

Reply:
I made most of the response readers/writers use `Identifier.foo` for most of the codeblocks. I'll do other parts as I encounter them (if there are more, I think response adapters are the places with the most actual codeblocks)